styleguide/components/41-organisms/timeline/_timeline.scss

```diff
@@ -0,0 +1,397 @@
+dl.timeline {
+  width: 100%;
+
+  * {
+    font-size: .8rem;
+   }
+
+  .timeline-slot {
+    @include theme('background-color', 'color-secondary', 'timeline-slot-default-background');
```

Comment:
Just keep it transparent. I don't see any indication in design that this has a background color.

Reply:
This has been done for subthemes... If we wish to use 2 other colors, this won't break anything.